### ISSUE TITLE
Exclude pattern is not make effect when setting in ~/.rubocop.yml for emacs flycheck.

### ISSUE THREAD

**Issue by zw963** [bbatsov/rubocop#5139]
Following is my settings begin part.

```yaml
AllCops:
  Include:
    - '**/Rakefile'
    - '**/config.ru'
  Exclude:
  - 'vendor/**/*'
  - 'spec/fixtures/**/*'
  - 'db/schema.rb'
  - 'db/migrate/*'
  - 'bin/bundle'
  - 'bin/rails'
  - 'bin/rake'
  - 'bin/setup'
  - 'bin/spring'
  - 'config/**/*'
  - 'script/**/*'
  - !ruby/regexp /old_and_unused\.rb$/

  TargetRubyVersion:
    2.3
```

And flycheck can auto use rubocop, no any config ( I don't remember set it)

When i open `db/schema.rb`, I still got the following error message:

```sh
Warning (flycheck): Syntax checker ruby-rubocop reported too many errors (4897) and is disabled.
```

**Comment by pocke** [comment]
RuboCop does not exclude a file that is specified by command line.
For example:

```yaml
# .rubocop.yml
AllCops:
  Exclude:
    - 'test.rb'
```

```ruby
# test.rb
foo "foo"
```

```bash
# Execution rubocop without arguments
$ rubocop 
Inspecting 0 files


0 files inspected, no offenses detected


# Execution rubocop with `test.rb`
$ rubocop test.rb
Inspecting 1 file
C

Offenses:

test.rb:2:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
foo "foo"
    ^^^^^

1 file inspected, 1 offense detected
```

And we can use `--force-exclusion` option to exclude specified files.

```bash
$ rubocop test.rb --force-exclusion
Inspecting 0 files


0 files inspected, no offenses detected
```


So, I think flycheck should specify the `--force-exclusion` option to rubocop command.
Can you open an issue to flycheck? 


And please use the [ISSUE_TEMPLATE.md](https://github.com/bbatsov/rubocop/blob/master/.github/ISSUE_TEMPLATE.md) for the next time 😉

**Comment by zw963** [comment]
Sorry, It added in newest version flycheck.

Thanks.

**Comment by zw963** [comment]
Hi，after discuss，seem like this is still a issue relation to rubocop，please see this https://github.com/flycheck/flycheck/issues/1368#issuecomment-349034683

**Comment by pocke** [comment]
Sorry for my late reply. I'll see the issue.

**Comment by pocke** [comment]
Will this problem is fixed in flycheck?

**Comment by zw963** [comment]
Sorry, forget to close this issue, i think it was fixed now.

Thanks
